spec/models/training/content_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Training::Content, type: :model do
```

Comment:
`Page`, `Question`, and `Video` are our models. `Content` is a shared module of common functionality. We already have specs for each model. An RSpec shared example/context would work well here perhaps.